jkube-kit/enricher/generic/src/main/java/org/eclipse/jkube/enricher/generic/openshift/RouteEnricher.java

```diff
@@ -54,7 +53,9 @@
 
     @AllArgsConstructor
     private enum Config implements Configs.Config {
-        GENERATE_ROUTE("generateRoute", "true");
+        GENERATE_ROUTE("generateRoute", "true"),
+        TLS_TERMINATION("termination",null),
+        INSECURE_EDGE_TERMINATION_POLICY("insecureEdgeTerminationPolicy",null);
```

Comment:
nit: since both these fields belong to the tls spec section, I think it may be clearer if both parameters were prepended with `TLS_` variable name and `tls` for key.

```suggestion
        TLS_TERMINATION("tlsTermination",null),
        TLS_INSECURE_EDGE_TERMINATION_POLICY("tlsInsecureEdgeTerminationPolicy",null);
```

Reply:
OK, sounds good. Tried to commit directly from this page ("commit suggestion"), but build failed because the signed-off footer was not present. What's the easiest way to resolve this?

Reply:
Well, these values need to be changed in docs and within the rest of the code too.

If you can't/ If you want, I can create an additional commit with the changes.

P.S. please in general try to create Pull Requests from a branch other than master it makes things much easier.

Reply:
OK, I am more than happy to let you do it, sorry for the trouble. Regarding your suggestion, I'm beginning to see what you mean - should probably find a good guide somewhere on how to contribute without actually generating more work for everybody else...

Reply:
No worries.

The main difference is that if you create another branch I can work directly on it with ease (master is protected, so in case I don't do it right the first time, I'll need to create a different PR - can't force push)

Reply:
OK, so just to make sure: Fork the repo, create branch on the forked repo and do changes there, right?

Reply:
Due to the master branch issue, I cannot commit changes there. New PR here https://github.com/eclipse/jkube/pull/411.
Please @sejacobsen check I didn't miss anything.